docs/ref/forms/api.txt

```diff
@@ -952,6 +975,29 @@ Attributes of ``BoundField``
 
         <label for="myFIELD">...</label><input id="myFIELD" type="text" name="my_field" required>
 
+.. attribute:: BoundField.initial
+
+    Use :attr:`BoundField.initial` to retrieve initial data for a form field.
+    It retrieves the data from :attr:`Form.initial` if present, otherwise
+    :attr:`Field.initial`, and evaluates if a callable. See
```

Comment:
Maybe... 

```suggestion
    It retrieves the data from :attr:`Form.initial` if present, otherwise
    trying :attr:`Field.initial`. Callable values are evaluated. See
```